### PR TITLE
tiup bench: modify command from --output to --output-dir

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Close https://github.com/pingcap/docs-cn/issues/10803

- Modify the flag from `--output` to `--output-dir`. The `--output` flag has been modified to `--output-dir` in this [tiup/211](https://github.com/pingcap/tiup/pull/211) PR, but the flag in the `tiup/tiup-bench.md` document is not updated. Since this change is before TiDB v4.0, this PR needs to cherry pick from v4.0 to v6.2. (issue: https://github.com/pingcap/docs-cn/issues/10803)
- Add the `--output-type=csv` in each command to generate CSV file (suggestions from JaySon-Huang)
- Remove the enable pprof step (suggestions from JaySon-Huang)

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tiup/pull/211

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-bench.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Available Commands:
 Flags:
       --check-all        运行所有的一致性检测
   -h, --help             tpcc 的帮助信息
-      --output string    准备数据时生成 csv 文件的目录
+      --output-dir string    准备数据时生成 csv 文件的目录
```
Comment (JaySon-Huang) [inline tiup/tiup-bench.md:66]: `--output-dir` 以及 `--tables` 不是 `tiup bench tpcc` 下的选项，只是 `tiup bench tpcc prepare` 下的选项。
这里可以把 66 行及 68 行去掉。

File: tiup/tiup-bench.md [hunk 2]
```diff
@@ -108,23 +108,23 @@ Flags:
     {{< copyable "shell-regular" >}}
 
     ```shell
-    tiup bench tpcc --warehouses 4 prepare --output data
+    tiup bench tpcc --warehouses 4 prepare --output-dir data
```
Comment (JaySon-Huang) [inline tiup/tiup-bench.md:111]: 最新的 `tiup bench`/`go-tpc` 在不连 mysql 实例下，只是生成数据到目录这个功能在经过一些迭代后，不能运作了。。 相关 issue https://github.com/pingcap/go-tpc/issues/100
```
>  tiup bench tpcc --warehouses 4 prepare --output-dir data
tiup is checking updates for component bench ...
A new version of bench is available:
   The latest version:         v1.10.3
   Local installed version:    v1.10.2
   Update current component:   tiup update bench
   Update all components:      tiup update --all

Starting component `bench`: /root/.tiup/components/bench/v1.10.2/tiup-bench tpcc --warehouses 4 prepare --output-dir data
Cannot open database, please check it (ip/port/username/password)
Error: dial tcp 127.0.0.1:4000: connect: connection refused
...
dial tcp 127.0.0.1:4000: connect: connection refused
```

Comment (JaySon-Huang) [reply tiup/tiup-bench.md:111]: 不过假如用户执行的时候，在 127.0.0.1:4000 上有 tidb 或 mysql，那么是可以运行的。文档改动先忽略这个报错的问题好了。

Comment (JaySon-Huang) [inline tiup/tiup-bench.md:111]:
```suggestion
    tiup bench tpcc --warehouses 4 prepare --output-dir data --output-type=csv
```

File: tiup/tiup-bench.md [hunk 3]
```diff
@@ -108,23 +108,23 @@ Flags:
...
     ```
 
 6. 为指定的表生成 CSV 文件：
 
     {{< copyable "shell-regular" >}}
 
     ```shell
-    tiup bench tpcc --warehouses 4 prepare --output data --tables history,orders
+    tiup bench tpcc --warehouses 4 prepare --output-dir data --tables history,orders
```
Comment (JaySon-Huang) [inline tiup/tiup-bench.md:119]:
```suggestion
    tiup bench tpcc --warehouses 4 prepare --output-dir data --output-type=csv --tables history,orders
```

File: tiup/tiup-bench.md [hunk 4]
```diff
@@ -108,23 +108,23 @@ Flags:
...
     ```
 
 7. 开启 pprof：
 
     {{< copyable "shell-regular" >}}
 
     ```shell
-    tiup bench tpcc --warehouses 4 prepare --output data --pprof :10111
+    tiup bench tpcc --warehouses 4 prepare --output-dir data --pprof :10111
     ```
```
Comment (JaySon-Huang) [inline tiup/tiup-bench.md:126]: 对于测试数据库性能，这一步不是普通在性能测试中会进行的步骤，建议直接删除掉。

File: tiup/tiup-bench.md (hunk 2 above)
Comment (JaySon-Huang) [reply tiup/tiup-bench.md]: 必须指定 --output-type=csv。
如果不指定 --output-type，那么会直接导入到 host:port 的数据库，而不是生成 csv 文件

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Review by Yui-Song — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**